app/controllers/courses_controller.rb

```diff
@@ -15,7 +15,9 @@ class CoursesController < ApplicationController
   def create
     slug_from_params if should_set_slug?
     @course =
-      Course.create(course_params.merge('passcode' => Course.generate_passcode))
+      Course.create(course_params.merge(
+        passcode: Course.generate_passcode,
+        type: ENV['default_course_type']))
```

Comment:
Currently, the default is handled by the database, but if `ENV['default_course_type']` is nil, this would create a course record that won't pass validation and won't get saved.

Let's do `type: ENV['default_course_type'] || 'ClassroomProgramCourse'`

That way, nothing will break for instances (including development instances) that leave their current application.yml as-is.